### PR TITLE
docs: add documentation for experimentalOriginDependencies

### PR DESCRIPTION
**Pull request by mschile**

### CODE REVIEW

**Review by mjhenkes — APPROVED** [review]

File: content/api/commands/origin.md [hunk 1]
```diff
@@ -408,18 +408,28 @@ packages and other files.
 
 <Alert type="warning">
 
-Using `import()` and `require()` within the callback requires version 5.15.0 or
-greater of the
+Using `import()` and `require()` within the callback requires requires enabling
```
Comment (AtofStryker) [inline content/api/commands/origin.md:411]:
```suggestion
Using `import()` and `require()` within the callback requires enabling
```

File: content/api/commands/origin.md [hunk 2]
```diff
@@ -408,18 +408,28 @@ packages and other files.
...
+_Using `require()` or `import()` to include dependencies requires enabling the
+`experimentalOriginDependencies` flag and using the latest version of
+`@cypress/webpack-preprocessor`._
+
+</Alert>
+
+<Alert type="warning">
+
+Using `require()` or `import()` within the callback from a `node_modules` plugin
+is not currently supported. We anticipate to add support with issue
```
Comment (chrisbreiding) [inline content/api/commands/origin.md:431]:
```suggestion
is not currently supported. We anticipate adding support with issue
```

File: content/api/commands/origin.md [hunk 3]
```diff
@@ -408,18 +408,28 @@ packages and other files.
 
 <Alert type="warning">
 
-Using `import()` and `require()` within the callback requires version 5.15.0 or
-greater of the
+Using `import()` and `require()` within the callback requires requires enabling
+the [`experimentalOriginDependencies`](/guides/references/experiments) flag in
+the Cypress configuration and using version 5.15.0 or greater of the
```
Comment (AtofStryker) [inline content/api/commands/origin.md:413]:
```suggestion
the Cypress configuration and using version `5.15.0` or greater of the
```

**Review by chrisbreiding — APPROVED** [review]

File: content/api/commands/origin.md [hunk 4]
```diff
@@ -408,18 +408,28 @@ packages and other files.
...
+is version 5.15.0 or greater.
 
 If using an older version of the webpack or a different preprocessor, you'll see
 an error that includes the following text:
 
-_Using require() or import() to include dependencies requires using the latest
-version of @cypress/webpack-preprocessor._
+_Using `require()` or `import()` to include dependencies requires enabling the
+`experimentalOriginDependencies` flag and using the latest version of
+`@cypress/webpack-preprocessor`._
```
Comment (AtofStryker) [inline content/api/commands/origin.md:424]:
```suggestion
`@cypress/webpack-preprocessor`.
```

Comment (AtofStryker) [reply content/api/commands/origin.md:424]: whoops thought I removed this one

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**